pkg/destroy/clusterapi/destroy.go

```diff
@@ -0,0 +1,36 @@
+package clusterapi
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/installer/cmd/openshift-install/command"
+	"github.com/openshift/installer/pkg/clusterapi"
+)
+
+// DestroyClusterAPIResources removes all cluster api artifacts that are not destroyed by
+// other destroy processes.
+func DestroyClusterAPIResources() error {
+	artifacts := []string{
+		// The Artifacts directory does not contain assets. These assets would be
+		// removed with a Purge and the directory would be automatically removed when all assets are gone.
+		// Remove the Artifacts Directory on cluster destroy
+		filepath.Join(command.RootOpts.Dir, clusterapi.ArtifactsDir),
+	}
+
+	for _, artifact := range artifacts {
+		if _, err := os.Stat(artifact); err == nil {
+			if !os.IsNotExist(err) {
+				logrus.Debugf("deleting artifact %s", artifact)
+				if err := os.RemoveAll(artifact); err != nil {
```

Comment:
If we're going to `rm -rf <workdir>/.clusterapi_output` anyway, maybe we can simplify this function to do just that? We can be more selective in the future if there is a need for it.